lib/linguist/languages.yml

```
@@ -5433,6 +5433,13 @@ Solidity:
   extensions:
   - ".sol"
   language_id: 237469032
+Solution:
```

I think this might be a bit too generic. How about:

```suggestion
Microsoft Visual Studio Solution:
```

This makes it a bit more consistent with "Microsoft Developer Studio Project" which we've already got.
```